radix-engine-interface/src/api/types/invocation/native.rs

```diff
@@ -28,7 +28,7 @@ pub enum NativeFn {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
 #[scrypto(TypeId, Encode, Decode, Describe)]
 pub enum NativeMethod {
-    AccessRules(AccessRulesMethod),
+    AccessRulesChain(AccessRulesChainMethod),
```

Comment:
With "Chain" suffix, suggest this just be renamed to `AccessRuleChain`

Reply:
Ah yes, you're right. For 2 I think we voted for `AccessCheck` (name for method is `add_access_check`) but don't quite like it (maybe something like `ComponentAuthGate`? to go along with `ComponentRoyaltyGate`). But okay with what you have now for now.

Reply:
Yeah, I like `AccessRule`, `AuthGate`, `AuthGateList` too. 

Will keep it as-is for now before we finalise on the names. :)